### ISSUE TITLE
get token key failed from google

### ISSUE THREAD

**Issue by Triggered0** [zlargon/google-tts#36]
Error: get token key failed from google

**Comment by Failure404** [comment]
Same issue here.
Not able to get media file via google-tts:get token key failed from google

**Comment by pavlukivan** [comment]
Google switched to a new API #35

**Comment by ncpierson** [comment]
This is a duplicate of #33

**Comment by freddiefujiwara** [comment]
let me try to fix  on https://github.com/zlargon/google-tts/pull/37

**Comment by ncpierson** [comment]
For anyone looking for an immediate solution, you can check out my comment I left in issue #35 : https://github.com/zlargon/google-tts/issues/35#issuecomment-737032391

I created a new package [google-translate-tts](https://github.com/ncpierson/google-translate-tts) which uses the new Translate API. Who knows if it will stick, but it fixed the immediate problems I was having.

**Comment by KimYou-Developer** [comment]
![IMG_20201204_130826](https://user-images.githubusercontent.com/67356294/101128287-e0569480-3631-11eb-9939-f8b6df301ec5.jpg)
So tried when I saw this

**Comment by freddiefujiwara** [comment]
This is my idea of api.js, but let me figure out how to merge it properly into api.js.

```JavaScript
const fetch = require('node-fetch');
module.exports = (async (text, lang, speed) => {
  const data =
    JSON.parse(JSON.parse((await (await fetch("https://translate.google.com/_/TranslateWebserverUi/data/batchexecute", {
      method: "POST",
      headers: {
        "content-type": "application/x-www-form-urlencoded"
      },
      body: "f.req=" + encodeURIComponent(JSON.stringify([
        [
          ["jQ1olc", JSON.stringify([text, lang, parseInt(speed) === 1 ? null : true, "null"]), null, "generic"]
        ]
      ]))
    })).text()).replace(/^\)]}'/, ''))[0][2])[0];
  return `data:audio/mpeg;base64,${data}`;
});
```

**Comment by freddiefujiwara** [comment]
Let me try to adopt ↑ at https://github.com/zlargon/google-tts/pull/38

**Comment by zlargon** [comment]
This issue has been fixed in PR https://github.com/zlargon/google-tts/pull/37 by @freddiefujiwara and has been released in version 0.0.6.
Thank you all. 🎉
